### ISSUE TITLE
Include VLC by default

### ISSUE THREAD

**Issue by kj7rrv** [linuxmint/linuxmint#542]
Celluloid does not support DVD menus, and apparently there is no plan to add the support back into MPV. Is it possible to include both Celluloid and VLC by default, and make VLC the default for DVDs and Celluloid the default for everything else? Not having proper DVD support out of the box seems like a significant issue.

**Comment by andreamtp** [comment]
Agree, also, given how excellent VLC is in term of media handling, why having audio player and video player, when with VLC you have the de-facto cross OS standard tool for reproducing media?
This will also make useless the request at installation time for `mint-media-codec`
